tests/test_embed/test_derived.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
Does `test_trampoline` make sense here? To leave it a little bit open (for the future) but not a widely as "derived".

Reply:
Renamed to `test_trampoline.py`